src/plone/api/content.py

```diff
@@ -18,9 +17,19 @@
 from zope.interface import Interface
 from zope.interface import providedBy
 
+import pkg_resources
 import random
 import transaction
 
+try:
+    pkg_resources.get_distribution('Products.Archetypes')
+except pkg_resources.DistributionNotFound:
+    class IBaseObject(Interface):
+        """ Products.Archetypes.interfaces.base.IBaseObject
```

Comment:
Not according to docstring conventions. Should be a single line (if short enough), no leading/trailing spaces.